editor/store/index.js

```diff
@@ -12,6 +12,10 @@ import {
 	getCurrentPostType,
 	getEditedPostContent,
 	getEditedPostTitle,
+	getLastMultiSelectedBlockUid,
+	getMultiSelectedBlocks,
+	getMultiSelectedBlockUids,
+	getSelectedBlock,
```

Comment:
Why are we exposing those selectors?

Reply:
Ah, I think at https://github.com/WordPress/gutenberg/pull/5021/files#diff-a6cb47a97748d7742a26ae134e49e8a1R76 I started down the path of using `query` but realized I have no access to dispatch which is necessary 😄